content/2016/10/18/versioning.rst

```diff
@@ -0,0 +1,45 @@
+public: no
+author: miriam
+tags: ['interview', 'sitepoint', 'podcast']
+image:
+  - src: '2016/versioning.png'
+summary: |
+  I joined Tim and David
+  for their great new `SitePoint`_ podcast —
+  `The Versioning Show`_.
+  We discussed going from lurker to speaker,
+  the importance of writing about what you’re learning,
+  stumbling into fame,
+  approaching new projects,
+  and unit testing in Sass.
+
+  .. _SitePoint: https://www.sitepoint.com/
+  .. _The Versioning Show: https://www.sitepoint.com/versioning-show-episode-8-miriam-suzanne/
+
+
+Versioning Show Ep.8: Miriam Suzanne
+====================================
+
+I joined Tim and David
```

Comment:
Perhaps display full name and link to their twitter accounts or Site Point author pages? 
https://www.sitepoint.com/author/tevko/ https://www.sitepoint.com/author/mdavidgreen/